specs/OAuth/OAuth2Manager.md

```diff
@@ -39,47 +39,6 @@ and OAuth 2.0 for Native Apps [RFC 8252](https://tools.ietf.org/html/rfc8252).
 
  ## Perform OAuth 2.0 (c++)
```

Comment:
We should mention that our oAuth2.0 API intentionally does not implement the implicit request because of so and so reason.

Reply:
Done.